File: 4.4.c
#include<stdio.h>
#include<ctype.h>
#include<string.h>
int main(){
    int V=0,c=0;
    char s [100];
    char v[]="aAeEiIoOuU";
    printf("Enter a string :\n");
    fgets(s,sizeof(s),stdin);
    for (int i=0;s[i]!='\0';i++){
        if (isalpha(s[i])){
            if ( strchr(v , s[i])){
                V++;
            }
            else{
                c++;
            }

        }
    }
   printf("Number of vowels: %d\n", V);
   printf("Number of consonants: %d\n", c);
   return 0;
}

File: 4.4.py
s=input("Enter a string:\n")
v=0
c=0
V="aAeEuUiIoO"
for i in s:
    if i.isalpha():
        if i in V:
            v+=1
        else:
            c+=1
print(f"Number of vowels: {v}")
print(f"Number of consonants: {c}")